Documentation/error-handling.md

```diff
@@ -0,0 +1,150 @@
+## Dialing In gRPC
+
+grpc.Dial is a function in the gRPC library that creates a virtual connection from the gRPC client to the gRPC server. 
+It takes a server URI (which can represent the name of a logical backend service, This URI could resolve to multiple actual addresses) 
```

Comment:
s/which can represent the name of a logical backend service, This URI could resolve to multiple actual addresses/which can represent the name of a logical backend service and could resolve to multiple actual addresses/